tests/misc_env/install_prereq.py

```diff
@@ -130,3 +132,22 @@ def enable_rhel_rpms(ceph, distro_ver):
     for repo in repos:
         ceph.exec_command(
             sudo=True, cmd='subscription-manager repos --enable={r}'.format(r=repo), long_running=True)
+
+
+def registry_login(ceph, distro_ver):
+    # login to this registry 'registry.redhat.io' on all nodes
+    # docker for RHEL 7.x and podman for RHEL 8.x
+    cdn_cred = get_cephci_config()['cdn_credentials']
+    user = cdn_cred.get('username')
+    pwd = cdn_cred.get('password')
+
+    conatiner = 'docker'
+    if distro_ver.startswith('8'):
+        conatiner = 'podman'
+
+    ceph.exec_command(cmd='sudo yum install -y {c}'.format(c=conatiner), long_running=True)
+
+    ceph.exec_command(cmd='sudo systemctl restart {c}'.format(c=conatiner), long_running=True)
```

Comment:
why are we restarting here?  
does podman have daemon?

Reply:
@XDsubhash sorry, I will make this restart only to docker, because I noticed some time docker service works only after restarting.